pycbc/waveform/parameters.py

```diff
@@ -441,7 +441,7 @@ def docstr(self, prefix='', include_label=True):
                 dtype=int, default=0,
                 description="Flag for generating sidebands")
 mode_array = Parameter("mode_array",
-                dtype=int, default=0,
+                dtype=int, default=None,
```

Comment:
`dtype=int` ???

Reply:
Yeah this should be a list I guess.

The default `None` makes sense to me.